tests/MJanssen/Provider/RoutingServiceProviderTest.php

```diff
@@ -389,4 +390,29 @@ public function testInvalidBefore()
         $routingServiceProvider->addRoute($app, $route);
 
     }
+
+    /**
+     *
+     */
+    public function testAddBeforeAfterMiddlewareByString()
+    {
+        $app = new Application();
+        $routingServiceProvider = new RoutingServiceProvider();
+        $route = $this->validRoute;
+
+        $mwClassName = 'FooMiddleware';
```

Comment:
Better to write it in full, $middlewareClassName, abbreviations just confuse people.